GitUI/NBugReports/BugReportInvoker.cs

```diff
@@ -101,6 +107,15 @@ public static void Report(Exception exception, bool isTerminating)
                 LogError(exception, isTerminating);
             }
 
+            if (exception is FileNotFoundException fileNotFoundException
+                && ReportFailToLoadAnAssembly(fileNotFoundException) == UserAction.ReLaunch)
+            {
+                ProcessStartInfo pi = new(Environment.ProcessPath, string.Join(" ", Environment.GetCommandLineArgs().Skip(1)));
```

Comment:
It'd be great to have some comments explaining what's going on here. E.g., what's in `Environment.GetCommandLineArgs()` and why we're skipping the first one.